pkgs/modules/docker/default.nix

```diff
@@ -102,18 +102,18 @@ let
   mobyGoPackagePath = "github.com/docker/docker";
   mobyVersion = "24.0.7+replit";
 
-  replit-moby = pkgs.buildGoPackage {
+  replit-moby = pkgs.buildGoModule {
     pname = "replit-moby";
-    version = mobyVersion;
+    version = "24.0.7+replit";
 
     src = pkgs.fetchFromGitHub {
       owner = "moby";
       repo = "moby";
       rev = "v24.0.7";
-      sha256 = "sha256-VUgsclXkoHHNT+GgYL7qiCV/4V3P9RZrT9BegMVYaRU=";
+      hash = "sha256-VUgsclXkoHHNT+GgYL7qiCV/4V3P9RZrT9BegMVYaRU=";
     };
 
-    goPackagePath = mobyGoPackagePath;
+    vendorHash = "";
```

Comment:
Needs to be filled in?

Reply:
I'm kind of confused as to why it wasn't a build failure, either via local `nix build` or CI here. How can this be made to be a build failure, as other required parameters are?

Reply:
_ah_. I was confused because when the value was absent, it was a statically analyzable error, so it warned me even though I was not building the `docker` module.

When the value was supplied but incorrect, it became a runtime error so it didn't warn me even though it was incorrect.

It didn't flag here because we stopped building the modules as part of PR gating since it was burning so much CPU for little benefit.

Reply:
And yes, when explicitly built,
```
       > vendor folder exists, please set 'vendorHash = null;' in your expression
```
so all is well.

Reply:
I did not do this moby upgrade correctly.

Reply:
#420'd actually.